packages/block-library/src/list-item/hooks/use-enter.js

```diff
@@ -19,71 +19,72 @@ import useOutdentListItem from './use-outdent-list-item';
 
```

Comment:
not sure what you did here but I think I copied this one to button block, so you think it can be improved there too?

Reply:
That doesn't have a mapping useSelect because there's no outdent/indent